test/Microsoft.AspNetCore.ResponseCaching.Tests/ResponseCachingMiddlewareTests.cs

```diff
@@ -554,25 +554,68 @@ public void FinalizeCachingHeaders_StoresCachedResponse_InState()
         }
 
         [Fact]
-        public void NormalizeStringValues_NormalizesCasingToUpper()
+        public void GetOrderCasingNormalizedStringValues_NormalizesCasingToUpper()
         {
             var uppercaseStrings = new StringValues(new[] { "STRINGA", "STRINGB" });
             var lowercaseStrings = new StringValues(new[] { "stringA", "stringB" });
 
-            var normalizedStrings = ResponseCachingMiddleware.GetNormalizedStringValues(lowercaseStrings);
+            var normalizedStrings = ResponseCachingMiddleware.GetOrderCasingNormalizedStringValues(lowercaseStrings);
 
             Assert.Equal(uppercaseStrings, normalizedStrings);
         }
 
         [Fact]
-        public void NormalizeStringValues_NormalizesOrder()
+        public void GetOrderCasingNormalizedStringValues_NormalizesOrder()
         {
             var orderedStrings = new StringValues(new[] { "STRINGA", "STRINGB" });
             var reverseOrderStrings = new StringValues(new[] { "STRINGB", "STRINGA" });
 
-            var normalizedStrings = ResponseCachingMiddleware.GetNormalizedStringValues(reverseOrderStrings);
+            var normalizedStrings = ResponseCachingMiddleware.GetOrderCasingNormalizedStringValues(reverseOrderStrings);
 
             Assert.Equal(orderedStrings, normalizedStrings);
         }
+
+        [Fact]
+        public void GetOrderCasingNormalizedStringValues_PreservesCommas()
+        {
+            var originalStrings = new StringValues(new[] { "STRINGA, STRINGB" });
+
+            var normalizedStrings = ResponseCachingMiddleware.GetOrderCasingNormalizedStringValues(originalStrings);
+
+            Assert.Equal(originalStrings, normalizedStrings);
```

Comment:
The example here https://github.com/aspnet/ResponseCaching/pull/34/files#diff-80295bfdc844391de2ffaeba04ef7843R613 shows a similar scenario split into 2 values in the StringValues, so this one doesn't get split into 2?